fenix/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -1443,6 +1443,23 @@
                 app:destination="@id/translationsDialogFragment"
                 app:popUpTo="@id/translationSettingsFragment"
                 app:popUpToInclusive="true" />
+            <action
+                android:id="@+id/action_translationSettingsFragment_to_neverTranslateFragment"
```

Comment:
Similar to #4226, please consider a naming pattern to indicate these are for preferences for future organization.

Reply:
Done